Zend/zend_builtin_functions.c

```diff
@@ -1480,6 +1483,8 @@ ZEND_FUNCTION(restore_exception_handler)
 		ZVAL_COPY_VALUE(&EG(user_exception_handler), tmp);
 		zend_stack_del_top(&EG(user_exception_handlers));
 	}
+
+	// TODO Change to void
```

Comment:
@carusogabriel if you are interested :)

Reply:
Some of them only work because I removed the false returns by promoting the warnings to error ^^"

Reply:
Yeah, I wanted to mention this, too :)